sklearn/_build_utils/__init__.py

```diff
@@ -6,15 +6,8 @@
 
 from __future__ import division, print_function, absolute_import
 
-HASH_FILE = 'cythonize.dat'
 DEFAULT_ROOT = 'sklearn'
 
-# WindowsError is not defined on unix systems
-try:
-    WindowsError
-except NameError:
-    WindowsError = None
```

Comment:
Do we need this? (Sorry if I missed something, but on a unix system, why would a `WindowsError` be raised?)